Tests/conf.json

```diff
@@ -5764,8 +5726,7 @@
         "get_original_email_-_ews-_test": "Issue 27571",
         "Trend Micro Deep Security - Test": "outsourced",
         "Microsoft Teams - Test": "Issue 38263",
-        "EWS Extension: Powershell Online V2 Test": "Issue 39008",
-        "O365 - EWS - Extension - Test": "Issue 39008",
```

Comment:
Where is the `O365 - EWS - Extension - Test` test?

Reply:
duplicate key, edited, thx